quarkus/config-api/src/main/java/org/keycloak/config/StorageOptions.java

```diff
@@ -167,6 +167,13 @@ public String getProvider() {
             .buildTime(true)
             .build();
 
+    public static final Option<String> STORAGE_DEPLOYMENT_STATE_RESOURCES_VERSION_SEED = new OptionBuilder<>("storage-deployment-state-version-seed", String.class)
+            .category(OptionCategory.STORAGE)
+            .description("Secret that serves as a seed to mask the version number of Keycloak in URLs. Need to be identical across all servers in the cluster. Will default to a random number generated when starting the server which is secure but will lead to problems when a loadbalancer without sticky sessions is used or nodes are restarted.")
+            .hidden()
```

Comment:
just to make sure: without #13387 merged, this option would still be shown even when it's tagged "hidden" here. Is this wanted?

Reply:
Yes